pkg/acs/calacs/acsrej/acsrej_do.c

```diff
@@ -395,6 +399,7 @@ int acsrej_do (IRAFPointer tpin, char *outfile, char *mtype, clpar *par,
                    "Total sky level (electrons)");
         PutKeyFlt (sg.globalhdr, "REJ_RATE", (float)nrej / texpt,
                    "Cosmic ray impact rate (pixels/sec)");
+        PutKeyFlt (sg.globalhdr, "FLASHDUR", cumFlashDur, "");
```

Comment:
This new keyword should have a comment to explain what it is; specifically, total/cumulative flash duration in seconds (or minutes or hours or mjd? ).

Reply:
That's fine.